### PR TITLE
cookie auth

### PR DESCRIPTION
**Pull request by dwinston**
for #423

### CODE REVIEW

File: nmdc_runtime/api/endpoints/users.py [hunk 1]
```diff
@@ -32,6 +35,43 @@
...
+        response.set_cookie(
+            key=key,
+            value=token_response[key.replace("user_", "")],
+            max_age=2592000,
+        )
+    return response
+
+
 @router.get("/orcid_authorize")
 async def orcid_authorize():
```
Comment (PeopleMakeCulture) [inline nmdc_runtime/api/endpoints/users.py:76]: @dwinston Can we get rid of /orcid_authorize?

Comment (dwinston) [reply nmdc_runtime/api/endpoints/users.py:76]: Definitely

File: nmdc_runtime/api/main.py [hunk 2]
```diff
@@ -356,10 +364,15 @@ async def get_versions():
         "\n\n"
         "Dependency versions:\n\n"
         f'nmdc-schema={version("nmdc_schema")}\n\n'
-        "<a href='https://microbiomedata.github.io/nmdc-runtime/'>Documentation</a>"
+        "<a href='https://microbiomedata.github.io/nmdc-runtime/'>Documentation</a>\n\n"
+        '<auth-action><img src="/static/ORCIDiD_icon128x128.png" height="18" width="18"/> '
+        f'<a href="https://orcid.org/oauth/authorize?client_id={ORCID_CLIENT_ID}'
+        "&response_type=code&scope=openid&"
+        f'redirect_uri={BASE_URL_EXTERNAL}/orcid_code">Login with ORCiD</a></auth-action>'
     ),
```
Comment (PeopleMakeCulture) [inline nmdc_runtime/api/main.py:372]: Are the <auth-action> tags necessary? It looks like they're currently just rendered as plain text. I removed them and was still able to authenticate.

File: nmdc_runtime/api/endpoints/users.py [hunk 3]
```diff
@@ -32,43 +35,28 @@
...
-async def orcid_authorize():
-    """NOTE: You want to load /orcid_authorize directly in your web browser to initiate the login redirect flow."""
-    return RedirectResponse(
-        f"https://orcid.org/oauth/authorize?client_id={ORCID_CLIENT_ID}"
-        "&response_type=token&scope=openid&"
-        f"redirect_uri={BASE_URL_EXTERNAL}/orcid_token"
-    )
-
-
-@router.get("/orcid_token")
```
Comment (PeopleMakeCulture) [inline nmdc_runtime/api/endpoints/users.py:45]: I removed this one also

**Review by PeopleMakeCulture — APPROVED** [review]

File: nmdc_runtime/api/main.py (hunk 2 above)
Comment (dwinston) [reply nmdc_runtime/api/main.py:372]: not necessary. they were an experimental hook for me to toggle what was displayed based on auth status.

**Pull request merged**